src/main/java/com/facebook/openwifirrm/modules/RRMScheduler.java

```diff
@@ -95,6 +97,60 @@ public void execute(JobExecutionContext context)
 		}
 	}
 
+	/**
+	 * Parses Linux cron spec (with seconds) into Quartz compatible cron spec.
+	 * Quartz does not allow setting both day of week and day of month. If valid
+	 * Quartz cron already we don't need to do anything. Otherwise we can't use
+	 * Quartz to even parse out the components as it will just throw a parse
+	 * error.
+	 *
+	 * @param linuxCron Linux cron with seconds
+	 *        (seconds minutes hours day_of_month month day_of_week [year])
+	 *
+	 * @throws IllegalArgumentException when a linux cron cannot be parsed
+	 *         into a valid Quartz spec
+	 * @return String a Quartz supported cron
+	 */
+	public static String parseIntoQuartzCron(String linuxCron) {
+		if (CronExpression.isValidExpression(linuxCron)) {
+			return linuxCron;
+		}
+
+		String[] split = linuxCron.split(" ");
+		if (split.length < 6 || split.length > 7) {
+			// Quartz cron MUST have at least 6 and no more than 7 fields
+			return null;
+		}
+
+		String dayOfMonth = split[3];
+		String dayOfWeek = split[5];
```

Comment:
nit: before this, also declare
```
final int MONTH_INDEX = 3;
final int WEEK_INDEX = 5;
```
so that you don't need to hardcode the indices multiple times (you're using them below)